.github/workflows/flatpak-builder.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+name: Pre-Flathub PR Workflow
```

Comment:
not sure why you need this.

FWIW flathub repositories get automatically build on new PR and on commits into the relevant branches.

Reply:
Mostly for my self, to shorten the feedback loop, and see this my PR will fail for other test.

Reply:
Please remove it. We have our own CI.